src/libPMacc/include/algorithms/math/floatMath/pow.tpp

```diff
@@ -43,6 +43,7 @@ struct Pow<float, float>
     HDINLINE result operator()(const float& base, const float& exponent)
     {
 #ifdef __CUDA_ARCH__ /*device version*/
+        /* @todo: CUDA has no pow(float, float) function yet but uses the C99 powf(float, float) */
```

Comment:
Side note: CUDa 6.5 also define `extern __host__ __device__ __cudart_builtin__ float    __cdecl pow(float, int) throw();` but in math_functions.h:8654 might be that it is not permanent in old cuda versions.